.github/workflows/build_check.yml

```diff
@@ -0,0 +1,18 @@
+name: Build Check
+
+on:
+  pull_request:
+
+jobs:
+  build:
```

Comment:
One nitpick though - maybe just under `build` you could add name like `name: Compile Project` so it just doesn't have the single `build` when you click into the action?

![image](https://user-images.githubusercontent.com/3514085/210630979-99bcd6f6-6106-44ed-b06f-60ef844696f2.png)

Reply:
Added names to steps, and combined the checks into one file!